README.md

```diff
@@ -1,9 +1,14 @@
 <div align="center">
-  <img src="https://use.expensify.com/assets/logo/download/expensify-app-logo_circular/expensify-app-logo_circular.png" width="64" height="64">
+    <a href="https://Expensify.cash">
+        <img src="https://raw.githubusercontent.com/Expensify/Expensify.cash/master/web/favicon.png" width="64" height="64" alt="Expensify.cash Icon">
```

Comment:
NAB not sure but can/should we be using a relative path for this?

Reply:
From my tests I couldn't use a local path, but this is somewhat a relative path as it should be using whatever the favicon from master is.